src/config/urls.py

```diff
@@ -11,7 +11,8 @@
 )
 
 urlpatterns = [
-    path("api/v1/", include("safe_apps.urls", namespace="v1")),
```

Comment:
I've changed the namespace here. Since we are going to have more applications under `v1` I've split that namespace in two: `safe-apps` and `chains`.

By having:
```python
path("api/v1/", include("safe_apps.urls", namespace="v1")),
path("api/v1/", include("chains.urls", namespace="v1")),
```

We get the following warning:
```shell
(urls.W005) URL namespace 'v1' isn't unique. You may not be able to reverse all URLs in this namespace
```

Reply:
I'm going to tackle the `safe-apps` namespace change in a different PR. For now I reverted that change here: 940bc16

Reply:
Created issue here: #89